decidim-admin/config/locales/es.yml

```diff
@@ -325,13 +323,13 @@ es:
         form:
           interpolations_hint: 'Sugerencia: Puede utilizar "%{name}" en cualquier parte del cuerpo o asunto y será reemplazado por el nombre del destinatario.'
         index:
-          confirm_delete: '¿Estás seguro de que quieres eliminar este boletín?'
```

Comment:
Careful with the locale changes. If that was done intentionally, what about creating a separate PR?